src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/unit/service/CruService.kt

```diff
@@ -0,0 +1,19 @@
+package uk.gov.justice.digital.hmpps.approvedpremisesapi.unit.service
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.service.CruService
+
+class CruService {
```

Comment:
I think the file and class name ought to include "Test"? Not sure if the test runner relies on that naming or not?

Reply:
Good spot, have amended.  The `Test` postfix is just convention, actual tests are discovered by JUnit via reflection looking for the @Test annotation.